phalcon/mvc/micro/lazyloader.zep

```diff
@@ -60,7 +60,11 @@ class LazyLoader
 		let definition = this->_definition;
 
 		if typeof handler != "object" {
-			let handler = new {definition}();
+			if(!class_exists(definition)) {
+				throw new Exception("Handler '" . definition . "' doesn't exist");
+			}
+
+			let handler = create_instance(definition);
```

Comment:
As a far as I know `new {definition}();` faster than `create_instance(definition);`. So I'm not sure we should change this after class_exists check. Could you elaborate on that

Reply:
I'm not sure why `new {definition}()` causes a segment fault, but it does even if the class exists when there is a syntax error in the file inside of `_object_and_properties_init`.

I ended up using `create_instance` since it is the same function that `Phalcon\Di` uses to create classes and that doesn't have a segment fault when loading classes with syntax errors.